docs/code-search/code-navigation/writing_an_indexer.mdx

```diff
@@ -121,14 +121,14 @@ This table is maintained as an authoritative resource for users, Sales, and Cust
 | --------------------- | ----------------- | ------ | ---------- | ---------------- | --------------- | ---------- | ---------------- | -------------------- |
 | Go                    | [scip-go]         | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    |
 | TypeScript/ <br /> JavaScript | [scip-typescript] | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    |
-| C/C++                 | [scip-clang]      | 🟡      | ✓          | ✓                | ✓               | ✓          | ✓                | ✗                    |
-| Java                  | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✓                | ✓                    |
-| Scala                 | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✓                | ✓                    |
-| Kotlin                | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✗                | ✓                    |
-| Rust                  | [rust-analyzer]   | 🟢      | ✓          | ✓                | ✓               | ✓*         | ✗                | ✓                    |
+| C/C++                 | [scip-clang]      | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                     |
+| Java                  | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓         | ✓                | ✓                    |
+| Scala                 | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓         | ✓                | ✓                    |
+| Kotlin                | [scip-java]       | 🟢      | ✓          | ✓                | ✓               | ✓         | ✗                | ✓                    |
+| Rust                  | [rust-analyzer]   | 🟢      | ✓          | ✓                | ✓               | ✓         | ✗                | ✓                    |
 | Python                | [scip-python]     | 🟢      | ✓          | ✓                | ✓               | ✓          | ✓                | ✓                    |
-| Ruby                  | [scip-ruby]       | 🟢      | ✓          | ✓                | ✓               | ✓          | ✗                | ✓                    |
-| C#                    | [scip-dotnet] Build tools (`.sln`, `.csproj`)     | 🟠      | ✓          | ✓                | ✓               | ✗          | ✓                | ✓                    |
```

Comment:
I think we said to keep C# either with an X for cross-repo until we have a real customer using it.

I also think we should not mark C# as beta since we have multiple customer in production using it